package.json

```diff
@@ -64,7 +52,38 @@
   },
   "husky": {
     "hooks": {
+      "pre-commit": "pretty-quick --staged && yarn lint",
       "pre-push": "yarn run test:cy -- --env configFile=local"
```

Comment:
it's not a comment to this PR but to not forget about it, imho pre-push should use start-server-and-test https://docs.cypress.io/guides/guides/continuous-integration.html#Basics so the tests can be run without stating manually the app. This command would be also useful for CI

Reply:
Done